### PR TITLE
add package.json -- cordova 7.0.0

### PR DESCRIPTION
**Pull request by tgensol**
Cordova 7.0.0 is now asking for a package.json file

### CODE REVIEW

**Comment by patrickbussmann** [comment]
See https://cordova.apache.org/news/2017/05/04/cordova-7.html.
And https://cordova.apache.org/docs/en/latest/guide/hybrid/plugins/#publishing-plugins.

Thanks :+1:

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "id": "com.verso.cordova.clipboard",
+    "platforms": [
+      "ios",
+      "android",
+      "wp8"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tgensol/CordovaClipboard.git"
```
Comment (patrickbussmann) [inline package.json]: @@tgensol please change to "VersoSolutions" instead "tgensol". :+1:

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "engines": [
+    {
+      "name": "cordova",
+      "version": ">=3.0.0"
+    }
+  ],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/tgensol/CordovaClipboard/issues"
```
Comment (patrickbussmann) [inline package.json]: @@tgensol please change to "VersoSolutions" instead "tgensol". :+1:

File: package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  "homepage": "https://github.com/tgensol/CordovaClipboard#readme"
```
Comment (patrickbussmann) [inline package.json]: @@tgensol please change to "VersoSolutions" instead "tgensol". :+1:

**Review by patrickbussmann — CHANGES_REQUESTED** [review]
Wrong repository url given!

**Comment by tgensol** [comment]
@patrickbussmann done thanks

**Comment by ibnclaudius** [comment]
Any idea when this will be released?

**Comment by janpio** [comment]
PR looks good, lgtm!

**Comment by ibnclaudius** [comment]
As the last commit was over 3 years ago and the project seems completely abandoned, I've created a new one with the fixes. Make good use! https://github.com/ibnclaudius/CordovaClipboard

**Review by mreis1 — APPROVED** [review]

**Pull request closed without merging**